.github/workflows/clang_non_smp.yml

```diff
@@ -53,5 +53,7 @@ jobs:
         $CHARM_HOME/bin/charmrun ++local ./pgm_cbench_benchmark +p2
         $CHARM_HOME/bin/charmrun ++local ./pgm_cdt_example
         $CHARM_HOME/bin/charmrun ++local ./pgm_cdt_example +p2
+        $CHARM_HOME/bin/charmrun ./jacobi_example 100 10
```

Comment:
Oh, the `netlrts` build needs `++local` here for now.

Reply:
Ah, I see, thanks!